src/cart/utils/memcheck-cart.supp

```diff
@@ -395,6 +395,8 @@
    Memcheck:Value8
    ...
    fun:*golang.org/*
+   fun:racecall
+   fun:racecallatomic
```

Comment:
This is tightening the definition so the suppression will match in *fewer* cases with this modification which I don't think is the intention here.

Reply:
How should it be defined?